config/foreman.migrations/20220406150414_foreman_google.rb

```diff
@@ -0,0 +1,4 @@
+old_answer = answers.delete('foreman::compute::gce')
+
+answers['foreman::plugin::google'] ||= old_answer || false
+answers['foreman::cli::google'] ||= old_answer || false
```

Comment:
Users can have disabled the CLI and if you then enable a CLI plugin it'll fail. I think this should take care of it.
```suggestion
answers['foreman::cli::google'] ||= answers['foreman::cli'] && old_answer || false
```

But what I'd prefer is to be more explicit in the logic, which is to enable it if the CLI and Google plugin are installed:
```suggestion
answers['foreman::cli::google'] ||= answers['foreman::cli'] && answers['foreman::plugin::google']
```

And copy the same to the Katello migration.